README.md

```diff
@@ -84,20 +98,22 @@ A full command reference can be found in [Command Reference.md](./docs/Command_R
 
 To work on Hokusai itself, set up your local development environment like so:
 
-- As above, install `python`, `pip`, `docker`, `docker-compose` and `git`.
+- As above, install `python`, `pipenv`, `docker`, `docker-compose` and `git`.
```

Comment:
Do expect `pyenv` and`pipenv` to be installed before running various development/packaging commands? I'm a bit confused why we're `pip install`ing these tools within the project Makefile if they're prerequisites here.

Reply:
Should we add `pyenv` here as well?

Reply:
_Technically_ `pyenv` isn't needed, just like `nvm` wouldn't be needed for a node project. That's just the mechanism by which we're recommending the installation of python. `pipenv` definitely is directly required though.